README.md

```diff
@@ -46,96 +61,40 @@ ready, just host your web app on your service of choice.
-  * `PUSHOVER_API_TOKEN` - Used to enable pushover notifications for Care Portal treatments, this token is specific to the application you create from in [Pushover](https://pushover.net/)
-  * `PUSHOVER_USER_KEY` - Your Pushover user key, can be found in the top left of the [Pushover](https://pushover.net/) site
-
-
-#### Core
-
-  * `DISPLAY_UNITS` (`mg/dl`) - Choices: `mg/dl` and `mmol`.  Setting to `mmol` puts the entire server into `mmol` mode by default, no further settings needed.
-  * `MONGO_COLLECTION` (`entries`) - The collection used to store SGV, MBG, and CAL records from your CGM device
-  * `MONGO_TREATMENTS_COLLECTION` (`treatments`) -The collection used to store treatments entered in the Care Portal, see the `ENABLE` env var above
-  * `MONGO_DEVICESTATUS_COLLECTION`(`devicestatus`) - The collection used to store device status information such as uploader battery
-  * `PORT` (`1337`) - The port that the node.js application will listen on.
-  * `SSL_KEY` - Path to your ssl key file, so that ssl(https) can be enabled directly in node.js
-  * `SSL_CERT` - Path to your ssl cert file, so that ssl(https) can be enabled directly in node.js
-  * `SSL_CA` - Path to your ssl ca file, so that ssl(https) can be enabled directly in node.js
-
-## Setting environment variables
 Easy to emulate on the commandline:
 
 ```bash
-    echo 'MONGO_CONNECTION="mongodb://sally:sallypass@ds099999.mongolab.com:99999/nightscout"' >> my.env
+echo 'CUSTOMCONNSTR_mongo="mongodb://sally:sallypass@mymongohost.com/db"' >> my.env
+echo 'CUSTOMCONNSTR_mongo_collection="sallyCGMCollection"' >> my.env
 ```
 
 From now on you can run using
 ```bash
-    $ env $(cat my.env) PORT=1337 node server.js
+$ env $(cat my.env) PORT=1337 node server.js
 ```
 
-Your hosting provider probably has a way to set these through their GUI.
-
-### Vagrant install
-
-Optionally, use [Vagrant](https://www.vagrantup.com/) with the
-included `Vagrantfile` and `setup.sh` to install OS and node packages to
-a virtual machine.
-
-```bash
-host$ vagrant up
-host$ vagrant ssh
-vm$ setup.sh
-```
-
-The setup script will install OS packages then run `npm install`.
-
-The Vagrant VM serves to your host machine only on 192.168.33.10, you can access
-the web interface on [http://192.168.33.10:1337](http://192.168.33.10:1337)
+Your hosting provider probably has a way to set these through their
+GUI.
 
 More questions?
 ---------------
 
 Feel free to [post an issue][issues], but read the [wiki][wiki] first.
 
-[issues]: https://github.com/nightscout/cgm-remote-monitor/issues
-[wiki]: https://github.com/nightscout/cgm-remote-monitor/wiki
+[issues]: https://github.com/rnpenguin/cgm-remote-monitor/issues
+[wiki]: https://github.com/rnpenguin/cgm-remote-monitor/wiki
```

Comment:
Fix points to old repository, the nightscout one is the correct repository to point to.